x-pack/libbeat/reader/etw/controller.go

```diff
@@ -0,0 +1,120 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build windows
+
+package etw
+
+import (
+	"errors"
+	"fmt"
+	"syscall"
+)
+
+// GetHandler queries the status of an existing ETW session to get its handler and properties.
+func (s *Session) GetHandler() error {
```

Comment:
This doesn't return a Handler so the naming seems off. It seems like the purpose is to attach to an existing session so perhaps the naming could indicate that if true.

Reply:
Fixed e71475a5bb4d71e5b4da8fd6b47a62f6ec102ed2